packages/core/src/styles/layout/layout.stories.ts

```diff
@@ -1774,9 +1775,9 @@ export function patternsContentSiteSingleRail() {
       </div>
       <div class="demo-content">
         <div cds-layout="grid cols@sm:4 gap:lg p:lg p@md:xxl container:xl container:center" cds-text="center">
-          <cds-card cds-layout="p:xl p@md:xxl">card</cds-card>
-          <cds-card cds-layout="p:xl p@md:xxl">card</cds-card>
-          <cds-card cds-layout="p:xl p@md:xxl">card</cds-card>
+          <cds-card> <div cds-layout="p:xl p@md:xxl">card</div> </cds-card>
```

Comment:
It should but you have a `private-host` element because the `card` extends the `panel`. The layout will be applied to the `host` instead to the `private-host`.

Reply:
Does this mean that this won't work for consumers who want to use the element to set up the layout?

```html
<cds-card cds-layout="vertical">
  <div cds-token"align:top"></div>
  <div cds-token"align:bottom"></div>
</cds-card>
```

Reply:
Layouts have some issue with aligning elements when its projected content. The DOM treats the elements in the component template as elements between  the projected content. Sort of in a way of flattening out the content

```html
<div class="template-div">
  <slot></slot>
</div>
```

```html
<cds-card cds-layout="vertical">
  <div class="template-div"> <!-- the projected items are not direct children of the card so flex/grid items dont work -->
    <div cds-token"align:top"></div>
    <div cds-token"align:bottom"></div>
 </div>
</cds-card>
```

Once CSS Sub grid is widely supported or Safari fixes display:content in a more consistent way we don't have many ways around this.